api/core/api.proto

```diff
@@ -97,14 +99,16 @@ message ListenResultRequest {
 //   "executionID": "__EXECUTION_ID__",
 //   "taskKey":     "__TASK_KEY__",
 //   "outputKey":   "__OUTPUT_KEY__",
-//   "outputData":  "{\"foo\":\"bar\"}"
+//   "outputData":  "{\"foo\":\"bar\"}",
+//   "tags":        ["executionX", "test"]
 // }
 // ```
 message ResultData {
-  string executionID = 1; // The unique identifier of the execution.
-  string taskKey = 2;     // The key of the executed task.
-  string outputKey = 3;   // The output's key from the returned task.
-  string outputData = 4;  // The output's data from the returned task, encoded in JSON.
+  string executionID = 1;   // The unique identifier of the execution.
+  string taskKey = 2;       // The key of the executed task.
+  string outputKey = 3;     // The output's key from the returned task.
+  string outputData = 4;    // The output's data from the returned task, encoded in JSON.
+  repeated string tags = 5; // List of tags associated to the task
```

Comment:
> "List of tags associated to the task"

This is not the task's tags but the execution's tags.

Should be updated to: "The list of tags associated to the execution"

Reply:
What about to rename `tags` to `executionTags`. It provide more context and match the `executionID` style.

Reply:
I like just `tags`, we are on the result of an execution already and actually maybe the `executionID` should just be `ID`

Reply:
As it is the gRPC API, I suggest to go with the most explicit names.
In the JS and Go libs, of course we will use only "tags" if it makes sense.
Also, all the variable are composed of 2 words. Let's follow the convention we used.

Reply:
`taskKey`, `outputKey` and `outputData` are composed with 2 words because they are nested data of the execution, and also because it kind of define the type of value needed here either a key or a data (that we use to symbolize json encoded payload). In the case of `tags` it's neither a nested data/object of the execution and we don't define the "type" otherwise we would call it something like `tagList`.

Really I don't see much the point of adding this. It is attached to a payload so we have the context already. We don't add `executionTaskKey` or `executionOutputKey` so there is no reason to do the same for the tags for me

Reply:
maybe name of _ResultData_ should be changed as _ExecutionResult_, replace _executionID_ with ID and just use _tags_.